### PR TITLE
chore: 🤖 Update changelog

### PR DESCRIPTION
**Pull request by ZedLi**
# Description
Updated changelog for DC and CLI version.

### CODE REVIEW

File: ui/desktop/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,22 @@
...
+### New
+
+- Upgrade CLI to v0.18.0 ([PR](https://github.com/hashicorp/boundary-ui/pull/2516))
+- Added support for notifications with transparent sessions ([PR](https://github.com/hashicorp/boundary-ui/pull/2336))
+- Added a user settings page for configuration options ([PR](https://github.com/hashicorp/boundary-ui/pull/2499))
+- Changed the limit of shown results to 250 and an indicator when cache daemon is loading ([PR](https://github.com/hashicorp/boundary-ui/pull/2500))
+
+### Bug Fixes
+
+- Fix to show username credentials before password ([PR](https://github.com/hashicorp/boundary-ui/pull/2494))
```
Comment (cameronperera) [inline ui/desktop/CHANGELOG.md:16]: Is this a bug fix? Seems like this would go under new as an improvement.

**Review by cameronperera — APPROVED** [review]
LGTM. Left a non-blocking comment.

File: ui/desktop/CHANGELOG.md (hunk 1 above)
Comment (ZedLi) [reply ui/desktop/CHANGELOG.md:16]: 🤷 I guess it depends on how you view the original intent, I'll move it over

File: ui/desktop/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,22 @@
...
+
+### New
+
+- Upgrade CLI to v0.18.0 ([PR](https://github.com/hashicorp/boundary-ui/pull/2516))
+- Added support for notifications with transparent sessions ([PR](https://github.com/hashicorp/boundary-ui/pull/2336))
+- Added a user settings page for configuration options ([PR](https://github.com/hashicorp/boundary-ui/pull/2499))
+- Changed the limit of shown results to 250 and an indicator when cache daemon is loading ([PR](https://github.com/hashicorp/boundary-ui/pull/2500))
+- Show username credentials before passwords ([PR](https://github.com/hashicorp/boundary-ui/pull/2494))
+
+### Bug Fixes
```
Comment (lisbet-alvarez) [inline ui/desktop/CHANGELOG.md:15]: Retry button for oidc authentication was also [fixed here](https://github.com/hashicorp/boundary-ui/pull/2512)

**Review by lisbet-alvarez — APPROVED** [review]
thanks for updating this doc!

File: ui/desktop/CHANGELOG.md (hunk 2 above)
Comment (ZedLi) [reply ui/desktop/CHANGELOG.md:15]: Good catch, I'll add that!

**Review by DhariniJeeva — APPROVED** [review]

**Pull request merged**